File: app/src/main/java/com/hussain/podcastapp/PodPlayApplication.java
package com.hussain.podcastapp;

import android.app.Application;

import com.crashlytics.android.Crashlytics;

import io.fabric.sdk.android.Fabric;

public class PodPlayApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Fabric.with(this, new Crashlytics());
    }
}

File: app/src/main/java/com/hussain/podcastapp/PodPlayApplication.kt
package com.hussain.podcastapp

import android.app.Application

import com.crashlytics.android.Crashlytics

import io.fabric.sdk.android.Fabric

class PodPlayApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Fabric.with(this, Crashlytics())
    }
}
